File: GreenopsBackendupdate/Models/Feedback.cs
namespace Greenops.Models
{
    public class Feedback
    {
        public int FeedbackId { get; set; }   // PRIMARY KEY

        public int UserId { get; set; }

        public string FeedbackContent { get; set; }

        public DateTime SubmittedDate { get; set; }
    }
}

File: GreenopsBackend_JavaSpring/src/main/java/com/greenops/entity/Feedback.java
package com.greenops.entity;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name = "Feedback")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Feedback {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "FeedbackId")
    private Integer feedbackId;

    @Column(name = "UserId", nullable = false)
    private Integer userId;

    @Column(name = "FeedbackContent", nullable = false)
    private String feedbackContent;

    @Column(name = "SubmittedDate", nullable = false)
    private LocalDateTime submittedDate;
}